server.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
+const app = express();
+const PORT = 3000;
+const { default: mongoose } = require('mongoose');
+const { mongoURI } = require('./config/mongodb-config.js');
+
+app.use(express.json());
+
+const authRoutes = require('./routes/authRoutes.js');
+const { validationRoutes } = require('./validationRoutes.js');
+const { errorHandler } = require('./middleware/errorMiddleware.js');
+const { ApiError } = require('./middleware/ApiError.js');
+const { passport } = require('./middleware/passport-middleware.js');
+//const { sequelize } = require('./config/squelize-config.js');
+const { postRouter } = require('./routes/postRouter.js');
+const { userRouter } = require('./routes/userRouter.js');
+
+app.use(passport.initialize());
+app.use('/api/auth', authRoutes);
+app.use('/',userRouter);
```

Comment:
for what do you use this router?

Reply:
this rout for check with pass "get me "   /me;